src/mixins/filing-template-mixin.ts

```diff
@@ -179,15 +181,17 @@ export default class FilingTemplateMixin extends Mixins(DateMixin, EnumMixin) {
 
     // add in Registration / Change of Registration data
     // *** FUTURE: change this to "if firm correction"
-    if (this.isCorrectedRegistration || this.isCorrectedChangeReg) {
+    if (
+      this.isCorrectedRegistration ||
+      this.isCorrectedChangeReg ||
+      this.isCorrectionFiling
```

Comment:
I think this is incorrect, since it will be True for BEN IA corrections (which don't have the data below).

If one of isCorrectedRegistration or isCorrectedChangeReg isn't True in your testing, it would be good to figure out why and fix that.

Reply:
Fixed the issue was in checking for the filing type before I set the filing type.

Reply:
Good find! Yes, I see the change in parseCorrectionFiling(). 👍